src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -530,6 +528,16 @@ public SymbolInfo GetSymbolInfo(ExpressionSyntax expression, CancellationToken c
 
                 return TypeFromVariable((SingleVariableDesignationSyntax)parent.Designation, cancellationToken);
             }
+            else if (expression is DeclarationExpressionSyntax declaration)
+            {
+                if (declaration.Designation.Kind() != SyntaxKind.SingleVariableDesignation)
+                {
+                    return SymbolInfo.None;
+                }
+
+                var symbol = GetDeclaredSymbol((SingleVariableDesignationSyntax)declaration.Designation, cancellationToken);
+                return new SymbolInfo(symbol);
```

Comment:
Thanks. I added it, but I'm having trouble to hit it with a test though.

---
In reply to: [95017393](https://github.com/dotnet/roslyn/pull/16153#discussion_r95017393) [](ancestors = 95017393)